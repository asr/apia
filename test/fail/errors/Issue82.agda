-- Testing the error when using an wrong ATP name. See the .flags
-- file.

{-# OPTIONS --exact-split              #-}
{-# OPTIONS --no-sized-types           #-}
{-# OPTIONS --no-universe-polymorphism #-}
{-# OPTIONS --without-K                #-}

postulate
  D   : Set
  _≡_ : D → D → Set
  a b : D

postulate p : a ≡ b
{-# ATP axiom p #-}

postulate foo : a ≡ b
{-# ATP prove foo #-}
