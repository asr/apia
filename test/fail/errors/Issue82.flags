--atp=e --with-e=xxx
