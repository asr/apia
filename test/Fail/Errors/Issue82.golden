ret > ExitFailure 1
out > Proving the conjecture in /tmp/test/Fail/Errors//Issue82/17-foo.fof
out >
err > apia: the ‘xxx’ command associated with E does not exist
err >
